models/aws_project.rb

```diff
@@ -94,14 +94,14 @@ def daily_report(date=(Date.today - 2), slack=true, text=true, rerun=false, verb
     end
   end
 
-  def weekly_report(date=Date.today - 2, slack=true, text=true, rerun=false, verbose=false)
+  def weekly_report(date=DEFAULT_DATE, slack=true, text=true, rerun=false, verbose=false)
     @verbose = false
     report = self.weekly_report_logs.find_by(date: date)
     msg = ""
     if report == nil || rerun
       record_instance_logs(rerun)
       get_latest_prices
-      usage = get_overall_usage(date == Date.today - 2 ? Date.today : date, true)
+      usage = get_overall_usage(date == DEFAULT_DATE ? Date.today : date, true)
```

Comment:
Similar question for this; we're using today's date if the `date` parameter is equal to `DEFAULT_DATE`?

Reply:
This is similarly checking if we are running the 'latest' report: if we are we want today's usage, if not then we want what was running on that given day.